_includes/already_have_an_account_banner.html

```diff
@@ -0,0 +1,24 @@
+<aside class="one-account-banner">
```

Comment:
The accessibility test failure here appears to be a real issue:

https://dequeuniversity.com/rules/axe/4.1/landmark-complementary-is-top-level

In other pages where we have a `one-account-banner`, we're including this next to (not part of) the `<main>` element. Ideally we'd be doing that with this one as well, but since this is part of the `2-col` template, I'm not sure how easy that is.

One idea is that we could do something similar to what we do with the other banners:

https://github.com/18F/identity-site/blob/30d57c1f0cfc8313079b409aee6f5fb6703d7430/_layouts/landing.html#L21-L23

Maybe have a different page property that controls whether to show this banner? Or enhance the existing `one_account_banner.html` to allow customizing it to show this version instead? Or maybe just directly target the page `{% if page.component.class == 'what-is-login' %}`?